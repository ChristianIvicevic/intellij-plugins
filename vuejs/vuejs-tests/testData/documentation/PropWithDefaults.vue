<script setup lang="ts">

export interface Props {
  items: string[]
}


const props = withDefaults(defineProps<Props>(), {
  items: () => [],
})

props.ite<caret>ms;
</script>

<template>
</template>