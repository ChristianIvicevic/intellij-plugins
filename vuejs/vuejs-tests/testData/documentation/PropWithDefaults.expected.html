<div class='definition'><pre>items: <span style="color:#000080;">string</span><span style="">[]</span></pre></div><table class='sections'><tr><td valign='top'><icon src='AllIcons.FileTypes.JavaScript'/>&nbsp;PropWithDefaults.vue</td></table>